Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/AbilData.xml

```diff
@@ -10149,6 +10147,21 @@
             </SectionArray>
         </InfoArray>
     </CAbilMorph>
+    <CAbilEffectTarget id="AP_DefilerSCBWConsumption">
+        <EditorCategories value="Race:Zerg,AbilityorEffectType:Units"/>
+        <Cost>
+            <Cooldown Location="Unit" TimeUse="10"/>
+        </Cost>
+        <TargetFilters value="Biological;Self,Enemy,Structure,Heroic,Missile,Uncommandable,Stasis,Dead,Hidden,Hallucination,Invulnerable"/>
+        <Range value="4"/>
+        <Effect index="0" value="AP_DefilerConsumption"/>
+        <CmdButtonArray index="Execute" DefaultButtonFace="AP_DefilerConsumption" Requirements="AP_NotHaveDefilerPredatoryConsumption"/>
+    </CAbilEffectTarget>
+    <CAbilEffectTarget id="AP_DefilerSCBWPredatoryConsumption" parent="AP_DefilerSCBWConsumption">
+        <TargetFilters index="0" value="Biological;Self,Structure,Heroic,Missile,Uncommandable,Stasis,Dead,Hidden,Hallucination,Invulnerable"/>
+        <Effect index="0" value="AP_DefilerPredatoryConsumption"/>
+        <CmdButtonArray index="Execute" DefaultButtonFace="AP_DefilerConsumption" Requirements="AP_HaveDefilerPredatoryConsumption"/>
+    </CAbilEffectTarget>

```

Comment:
lacks range actor:
![Snímek obrazovky pořízený 2024-05-19 11-56-06](https://github.com/Ziktofel/Archipelago-SC2-data/assets/3447808/639d98b4-522e-42a3-89fa-48bac60e5f14)
compared to this:
![Snímek obrazovky pořízený 2024-05-19 11-59-13](https://github.com/Ziktofel/Archipelago-SC2-data/assets/3447808/7b400194-3cae-4001-920e-e774509ef872)
Also the graphic effect/sound is missing (See actors - check references towards Infestor consume)

Reply:
Thanks for spotting that! Fixed.